RELEASE_NOTES.md

```diff
@@ -94,7 +130,7 @@ The following items are new since Enterprise Chef 11.2.1 and/or are changes from
   * `chef-server-ctl` replaces `private-chef-ctl` though
     `private-chef-ctl` will also work in CS12.
   * Several commands added related to the management of users and
-    organizations, allowing allowing management of organizations without the management console
```

Comment:
We used to take such a firm stance on `allowing`, I'm glad we're easing up on it.